erroranalysis/erroranalysis/_internal/constants.py

```diff
@@ -207,3 +210,10 @@ class ErrorCorrelationMethods(str, Enum):
     Metrics.MICRO_F1_SCORE,
     Metrics.ACCURACY_SCORE,
     Metrics.ERROR_RATE]
+
+
+object_detection_metrics = [
+    Metrics.MEAN_AVERAGE_PRECISION,
```

Comment:
Shouldn't we just start with error rate by calculating whether images get detected correctly or not in error analysis? These metrics would then just be computed on top of those split nodes as additional options, but we would still split the tree on whether images were detected correctly.

Reply:
Great point!